server/plugins/messageStorage/sqlite.ts

```diff
@@ -190,13 +185,11 @@ class SqliteMessageStorage implements ISqliteMessageStorage {
 		]);
 	}
 
-	/**
-	 * Load messages for given channel on a given network and resolve a promise with loaded messages.
-	 *
-	 * @param network Network - Network object where the channel is
-	 * @param channel Channel - Channel object for which to load messages for
-	 */
-	async getMessages(network: Network, channel: Channel): Promise<Message[]> {
+	async getMessages(
+		network: Network,
+		channel: Channel,
+		nextID: () => number
```

Comment:
it's not immediately clear to me what nextID is. Can we either re-add a comment or give it a better name? In this case, it seems like it's not the nextId so much as the new Id?

Reply:
well, function that if called gives you the next ID.

What's your favorite bikeshed color?
I don't care much, that was what my brain came up with in a pitch but I'm happy to adapt to whatever you feel works better

Reply:
I can certainly try a comment, but if you have a descriptive name in mind that might be better

Reply:
I see why we passed the entire `client` in before...

What if we added an `incrementId` function to Client and passed it into the MessageStorage in `server/client.ts`? Then we can leave `getMessages` as is.

Reply:
How about we call it gen_unique_msg_id() ?

Descriptive enough?

Reply:
I really really don't want the storage backends to have access to the client, that violates all kinds of layering so I'd rather not keep getmessages as is.

Reply:
The reason I'm passing it to this function is that this is the only place that thing gets used.

Yes, you could pass it into the constructor of the backend, but it seems very strange to pass it there, After all nothing but a single function uses it, so why add it to the state when we can use a param?